cleanlab_studio/studio/trustworthy_language_model.py

```diff
@@ -709,7 +709,7 @@ class TLMOptions(TypedDict):
         and helping catch answers that are obviously incorrect/bad for a prompt asking for a well-defined answer that LLMs should be able to handle.
         Setting this to False disables the use of self-reflection and may produce worse TLM trustworthiness scores, but will reduce costs/runtimes.
 
-        log (List[str], default = None): optionally specify additional logs or metadata to return.
+        log (List[str], default = []): optionally specify additional logs or metadata to return.
```

Comment:
Wouldn't the default still be None since log is specified as NotRequired?

Reply:
The defaults here are a little confusing, as they are all not actually specified in the client side code (eg. `num_candidate_responses (int, default = 1)` above is not explicitly specified but rather it is the setting we have for the `medium` preset.

I'm choosing to state `[]` as the default instead of `None` to prevent users from specifying `options={"log": None}` since that is an invalid input